runtime/src/iree/base/config.h

```diff
@@ -253,4 +253,9 @@ typedef IREE_DEVICE_SIZE_T iree_device_size_t;
 #define IREE_VM_EXT_F64_ENABLE 0
 #endif  // !IREE_VM_EXT_F64_ENABLE
 
+#if !defined(IREE_VMVX_UBSAN_CHECKABLE_ENABLE)
+// Exposes VMVX kernels to UBSAN checking, else disable UBSAN checking.
+#define IREE_VMVX_UBSAN_CHECKABLE_ENABLE 0
```

Comment:
this would be IREE_VM_ instead of vmvx - the vm is used by vmvx but also for running host code and this applies to both

Reply:
(otherwise LGTM and thanks for doing this!)